docs/options.md

```diff
@@ -65,6 +65,7 @@ const defaultOptions = {
   code: {
     // NEW
     es5: false,
+    exportEsm: false,
```

Comment:
maybe just esm?

also I was wondering, it’s not a problem if both es5 and esm are combined, esm export can be limited to es5 syntax too, so they are indeed orthogonal…

Reply:
>maybe just esm? 

That makes sense, follows the naming convention of the es5 option

>also I was wondering, it’s not a problem if both es5 and esm are combined, esm export can be limited to es5 syntax too, so they are indeed orthogonal…

Correct me if I am wrong, but at default es5 is false, so it uses es6. If we assume that many are using the default, it would be a breaking change if we make the rule that es6 exports in ESM. That is what happened while I was developing, many of the tests failed because they didn't specify es5 true, meaning es6 + ESM and then the tests failed because the moduleCode was still using `requireFromString` which did not work because the code is now a module so I had to use `importFromStringSync`. That is why I don't think they can be combined?

Reply:
Sorry, by “combined” I meant that it will be ok if a user uses both options. These should be separate options in ajv, you are right.

Reply:
Okay cool